### ISSUE TITLE
Add support for .esterc

### ISSUE THREAD

**Issue by grabbou** [grabbou/este-cli#1]

**Comment by AugustinLF** [comment]
Did you have any idea/plans to make that work? I already have a few needs, for this, considering that my app's structure slightly diverged from este on some points (location of files, and things like that).

What I need right now, is:
- The possibility to disable the `translate` function when generating a new feature (I significantly changed the way I handle the messages, and I don't think it will be easy for me to port this generation to my architecture (I'll obviously do it, but it's not a priority).
- Change the default location of some files (I moved the `src/client/state.js`), and I create all new features in a `feature` folder. This should be really simple to implement, the question is more how do we want to be able to configure that.

What I thought would of course be a json-like file, with options (never necessary, all should have a default value). It would be something like that:

```javascript
{
  "statePath": "app/", // default ""
  "featurePath": "feature/" // default ""
}
```

And then, instead of doing `var statePath = path.join(options.rootFolder, 'state.js');` we would do something like `var statePath = path.join(options.rootFolder, opts.getStatePath(), 'state.js');`.

I know that it might be a priority for you, but it's blocking for me, I have to change the paths in the source files, which is obviously not a sustainable solution.

**Comment by grabbou** [comment]
Yes, this is on the radar. Need to update after new flux is there and once we are fully compatible, will implement config file as well. In the meantime, please create issue for every feature you think CLI should read from config (different folder structure, some default options etc?)

**Comment by grabbou** [comment]
I'll try to implement ability to override options by the end of the week ;)

**Comment by grabbou** [comment]
Most likely I'll give you a skeleton that's easy to tweak so you can implement more advanced features on top of it as well ;)
